csgo/hooking/manager.cpp

```diff
@@ -67,6 +67,11 @@ bool c_hooks::hook( ) {
 		return false;
 	}
 
+	if (!m_directx.hook_method(hook::idx::RESETEX, &hook::ResetEx)) {

```

Comment:
NOTE: I haven't checked if that fails with d3d9ex disabled but in that case I suppose we could make it `if (!hook(reset) && !hook(resetex))` then fail.